src/pages/dashboard/home.tsx

```diff
@@ -124,37 +124,4 @@ export default () => {
       </Content>
     </SidebarLayout>
   );
-
```

Comment:
Don't we still need this until the event?

Reply:
I'm not sure what you're pointing at here... I haven't actually removed anything.

Reply:
It's exactly the same stuff that's a few lines above in the sidebar layout. I didn't delete it initially to avoid conflicts with what others were doing.